docs/content/topics/operators/storage.md

```diff
@@ -9,189 +9,169 @@ aliases:
-
-## Supporting Brigade Storage
-
-Only certain volume plugins _can_ support Brigade. Specifically, **a volume driver
-must be readWriteMany** in order for Brigade to use it. At the time of writing
-very few VolumePlugins support the `readWriteMany` access mode. Ensure that your
-volume plugin can support `readWriteMany`
-([table](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes))
-or that you're able to use [NFS](#using-an-nfs-server).
-
-Only the following volume drivers are tested:
-
-- Minikube's 9P implementation
-- Azure's AzureFile storage
-- NFS
+```yaml
+worker:
+  workspaceStorageClass: nfs
+```
 
-We believe Gluster will work, but it's untested.
+[PersistentVolume]: https://kubernetes.io/docs/concepts/storage/persistent-volumes/
+[Access Modes]: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes
+[storage class]: https://kubernetes.io/docs/concepts/storage/storage-classes/
+[NFS]: https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/tree/master/deploy/helm
+[Azure File]: https://azure.microsoft.com/en-us/services/storage/files/
+[AKS]: https://azure.microsoft.com/en-us/services/kubernetes-service/
+[Azure Disk]: https://azure.microsoft.com/en-us/services/storage/disks/
+[Helm chart values]: https://github.com/brigadecore/brigade/blob/v2/charts/brigade/values.yaml
 
-## Examples
+### Enabling Worker storage
 
-### Using an NFS Server
+To enable shared Worker storage, set `useWorkspace` to `true` under the
+`workerTemplate` section on the [project configuration file][project file]
+(usually `project.yaml`) for a Project. For example, here is the relevant bit
+of configuration from the [10-shared-workspace example project]:
 
-As Brigade uses storage for caching and short-term file sharing, it is often convenient
-to use storage backends that are optimized for short-term ephemeral storage.
-
-NFS (Network File System) is one protocol that works well for Brigade. You can
-use the [NFS Server Provisioner](https://github.com/helm/charts/tree/master/stable/nfs-server-provisioner)
-chart to easily install an NFS server.
+```yaml
+spec:
+  workerTemplate:
+    useWorkspace: true
+```
 
-```console
-$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/
-$ helm install --name nfs stable/nfs-server-provisioner
+Each Worker Job requiring access to this workspace must then be configured with
+a filepath value designating where the workspace should be mounted within the
+Job's container. (Note that this may be the Job's `primaryContainer` and/or one
+or more of a Job's `sidecarContainer`(s), if applicable). This filepath value
+is assigned to the `workspaceMountPath` field on each applicable Job container.
+
+In the example `brigade.js` script below, the Job named "first-job" is
+configured with the `workspaceMountPath` value set to `/share` on both its
+primary container and its sidecar container, "sidecar". The sidecar container
```

Comment:
Using a sidecar in this example might unnecessarily distract from what we're trying to get across.

Showing two jobs that execute in serial, with the first writing a message to a file and the second cat'ing the file might be more straightforward for readers.

Reply:
Perhaps true.  I just noticed that the example shared workspace project does what you've mentioned (two jobs in serial) and there didn't seem to be any examples showing configuring a Job's sidecar container as well, so I wanted to demonstrate that knowledge here.  However, no super strong opinion.  If we can get an example up (or maybe update the aforementioned example showing sidecar config), then I'd be cool with changing this up per your recommendation.  WDYT?

Reply:
Yes. Let's add a separate sidecar example and maybe we mention sidecars in the scripting guide.

Reply:
fyi, this is the only unresolved issue I can spot.

Reply:
Oh thanks, just noticed this.  Will update next.

Reply:
Updated; created separate issue tracking sidecar container docs/example.